### PR TITLE
add: Spooky Rory

### PR DESCRIPTION
**Pull request by AshtakaOOf**
Idk how to do the spooky Halloween jumpscare lol

Signed-off-by: Adrien <66513643+AshtakaOOf@users.noreply.github.com>

### CODE REVIEW

**Review by Scrumplex — APPROVED** [review]

File: launcher/resources/backgrounds/backgrounds.qrc [hunk 1]
```diff
@@ -8,8 +8,10 @@
         <file alias="rory">rory.png</file>
         <file alias="rory-xmas">rory-xmas.png</file>
         <file alias="rory-bday">rory-bday.png</file>
+        <file alias="rory-spooky">rory-spooky.png</file>
```
Comment (santiagocezar) [inline launcher/resources/backgrounds/backgrounds.qrc:11]: This should end with `-ween`

File: launcher/resources/backgrounds/backgrounds.qrc [hunk 2]
```diff
@@ -8,8 +8,10 @@
...
         <file alias="rory-flat">rory-flat.png</file>
         <file alias="rory-flat-xmas">rory-flat-xmas.png</file>
         <file alias="rory-flat-bday">rory-flat-bday.png</file>
+        <file alias="rory-flat-spooky">rory-flat-spooky.png</file>
```
Comment (santiagocezar) [inline launcher/resources/backgrounds/backgrounds.qrc:15]: and yeah this too

File: launcher/ui/MainWindow.cpp [hunk 3]
```diff
@@ -1564,7 +1564,7 @@ void MainWindow::setCatBackground(bool enabled)
         QString cat = APPLICATION->settings()->get("BackgroundCat").toString();
         if (non_stupid_abs(now.daysTo(xmas)) <= 4) {
             cat += "-xmas";
-        } else if (cat == "kitteh" && non_stupid_abs(now.daysTo(halloween)) <= 4) {
+        } else if (non_stupid_abs(now.daysTo(halloween)) <= 4) {
             cat += "-ween";
```
Comment (santiagocezar) [inline launcher/ui/MainWindow.cpp]: or the suffix could be `-spooky` instead, which I think seems nicer

**Review by santiagocezar — CHANGES_REQUESTED** [review]

File: launcher/resources/backgrounds/backgrounds.qrc (hunk 1 above)
Comment (Scrumplex) [reply launcher/resources/backgrounds/backgrounds.qrc:11]: oops

**Comment by AshtakaOOf** [comment]
AAAAAAh I pushed the wrong branch noooo
I wanted to add copr to the readme

**Review by DioEgizio — APPROVED** [review]

**Pull request merged**